nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-nar-utils/src/main/java/org/apache/nifi/nar/NarClassLoader.java

```diff
@@ -116,10 +122,13 @@
  * Maven NAR plugin will fail to build the NAR.
  * </p>
```

Comment:
I think the javadoc of this class (which describes the NAR structure / java classloading in detail) should be updated with the concept of the native lib handling too.

Usage of `-Djava.library.path` system property also needs to be documented (the Admin Guide may be the right place for it).
Updating the description of the properties having `dynamicallyModifiesClasspath(true)` also needed I think.

Reply:
I'll update the docs.